midas/query.py

```diff
@@ -0,0 +1,59 @@
+from sqlalchemy.ext.declarative.api import DeclarativeMeta
+import sqlalchemy
+
+from .session_creator import session
+
+
+class Query:
```

Comment:
You should use a different name, so there won't be mixups between your query and sqlalchemy's.